tests/lmcma_test.cpp

```diff
@@ -0,0 +1,15 @@
+
+ //Created by o
+
+
+#include <ensmallen.hpp>
+#include "catch.hpp"
+#include "test_function_tools.hpp"
+
+using namespace ens;
+using namespace ens::test;
+
+TEST_CASE("LMCMAInitTest", "[LMCMATest]")
+{
+  LMCMA c(1);
+}
```

Comment:
I imagine you aren't quite done with the test suite yet, but if you could perhaps replicate some of the other tests used for arbitrary optimizers, I think that would be sufficient here. :+1: